keyboards/qvex/lynepad2/info.json

```diff
@@ -0,0 +1,69 @@
+{
+    "manufacturer": "QVEX Tech",
+    "keyboard_name": "Lynepad",
+    "maintainer": "QVEX-Tech",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": [ "F0", "F1", "F4", "F5"],
+        "rows": [ "C7", "F7" ,"F6", "F6", "F6"]
```

Comment:
You have F6 defined multiple times here.  These are the encoders?  If so, you can replace these with "no_pin" or null

Reply:
Yes, these are the "virtual" rows for the encoder push buttons and tilt switches. I have replaced them with null.